src/pages/Main.js

```diff
@@ -14,29 +15,32 @@ import NpsSurvey from 'src/components/NpsSurvey'
 import { AuthenticatedCookieSetter } from 'src/components/runtime-common'
 import ServiceAlerts from 'src/components/ServiceAlerts'
 import SupportRequest from 'src/components/SupportRequest'
+import { getOidcConfig } from 'src/libs/auth'
 import { PageViewReporter } from 'src/libs/events'
 import { LocationProvider, PathHashInserter, Router, TitleManager } from 'src/libs/nav'
 
 
 const Main = () => {
-  return h(LocationProvider, [
-    h(PathHashInserter),
-    h(CookieWarning),
-    h(ReactNotification),
-    h(ImportStatus),
-    h(ServiceAlerts),
-    h(Favicon),
-    h(IdleStatusMonitor),
-    h(ErrorWrapper, [
-      h(TitleManager),
-      h(FirecloudNotification),
-      h(AuthenticatedCookieSetter),
-      h(AuthContainer, [h(Router)])
-    ]),
-    h(PageViewReporter),
-    h(SupportRequest),
-    h(NpsSurvey),
-    h(ConfigOverridesWarning)
+  return h(AuthProvider, getOidcConfig(), [
+    h(LocationProvider, [
+      h(PathHashInserter),
+      h(CookieWarning),
+      h(ReactNotification),
+      h(ImportStatus),
+      h(ServiceAlerts),
+      h(Favicon),
+      h(IdleStatusMonitor),
+      h(ErrorWrapper, [
+        h(TitleManager),
+        h(FirecloudNotification),
+        h(AuthenticatedCookieSetter),
+        h(AuthContainer, [h(Router)])
```

Comment:
Since we are not using the context, and don't want others to do so, another option could be to do something like:
```javascript      
       h(AuthProvider, getOidcConfig(), [h(AuthContextSetter)]),
       h(AuthContainer, [h(Router)])
```

Reply:
I like this! Good idea

Reply:
I did this